android/books/src/main/java/kmp/android/books/navigation/BooksDestinations.kt

```diff
@@ -0,0 +1,12 @@
+package kmp.android.books.navigation
+
+import androidx.annotation.StringRes
+
+sealed class BooksDestinations(val route: String, @StringRes val titleRes: Int) {
```

Comment:
This feels a little bit awkward, I saw there are three sealed classes like this, but they do not share the code, instead it is copied. You could create an Interface with route and titleRes to ensure all the destinations are the same (also the naming is inconsistent, sometimes there is `s` at the end and sometimes not).
In some projects we have nicer navigation (MujUp, SkySeat, UniquePint, ClashOfTheStars, ...) which we could use here 🙂

Reply:
Done😀